src/proto/series.c

```diff
@@ -304,7 +304,7 @@ static int proto_series_led_level_row(monome_t *monome, uint_t x_off,
 static int proto_series_led_level_col(monome_t *monome, uint_t col,
 		uint_t y_off, size_t count, const uint8_t *data) {
 	uint_t i, chunks;
-	uint8_t masks[2];
+	uint8_t masks[16];
```

Comment:
@tehn what exactly are these changes for?

Reply:
fixes `gcc` error

```
/home/tehn/local/serialosc/third-party/libmonome/src/proto/series.c:298:26: error: writing 1 byte into a region of size 0 [-Werror=stringop-overflow=]
  298 |                 masks[i] = reduce_levels_to_bitmask(&data[i*8]);
```